tests/em/static/test_SPjvecjtvecadj.py

```diff
@@ -117,3 +117,11 @@ def test_clears():
     sim.qMap = maps.ExpMap()
     assert sim.deleteTheseOnModelUpdate == ["_Jmatrix", "_gtgdiag"]
     assert sim.clean_on_model_update == []
+
+
+# test deprications
+def test_deprications():
+    with pytest.warns(FutureWarning):
```

Comment:
I am not sure why this is failing... it passed prior to mering #1430. Any insights or help would be appreciated!

Reply:
The reason why this is failing is because the test file is already importing `spontaneous_potential` outside the test function. Since the module was already imported, the warning won't be raised (code in `spontaneous_potential/__init__.py` won't be rerun). I'll fix the test file so the test passes.

Reply:
Ah, thanks @santisoler!